### PR TITLE
feat(django): Add example for Django

### PR DESCRIPTION
**Pull request by mschlumpp**
Adds an example of a Django project with bjoern as a WSGI server.

### CODE REVIEW

File: http-python3.12-django5.0/Kraftfile [hunk 1]
```diff
@@ -0,0 +1,7 @@
+spec: v0.6
+
+runtime: index.unikraft.io/official/python:3.12
```
Comment (nderjung) [inline http-python3.12-django5.0/Kraftfile:3]:
```diff
- runtime: index.unikraft.io/official/python:3.12
+ runtime: python:3.12
```

File: http-python3.12-django5.0/README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Python Django web application
+
+To run this example, first [install the `kraft` CLI tool](https://unikraft.org/docs/cli).
+
+Then, clone this repository and `cd` into this directory.
+To deploy this application on KraftCloud, invoke:
+
+```
+kraft cloud deploy -M 512 -p 443:80 .
```
Comment (nderjung) [inline http-python3.12-django5.0/README.md:9]: Can we switch this to `8080`, most other examples/catalog entries are using this (we should switch those that aren't), and `8080` helps with local development.

File: http-python3.12-django5.0/main.py [hunk 3]
```diff
@@ -0,0 +1,5 @@
+import bjoern
+import helloworld.wsgi
+
+bjoern.run(helloworld.wsgi.application, "0.0.0.0", 80)
```
Comment (nderjung) [inline http-python3.12-django5.0/main.py:4]:
```diff
- bjoern.run(helloworld.wsgi.application, "0.0.0.0", 80)
+ bjoern.run(helloworld.wsgi.application, "0.0.0.0", 8080)
```

File: http-python3.12-django5.0/requirements.txt [hunk 4]
```diff
@@ -0,0 +1,2 @@
+Django==5.0.1
```
Comment (nderjung) [inline http-python3.12-django5.0/requirements.txt:1]: Is `Django` meant to be capitalized?

File: http-python3.12-django5.0/README.md (hunk 2 above)
Comment (mschlumpp) [reply http-python3.12-django5.0/README.md:9]: I really dislike this whole port mess. Port 80 is the standard HTTP port...

File: http-python3.12-django5.0/requirements.txt (hunk 4 above)
Comment (mschlumpp) [reply http-python3.12-django5.0/requirements.txt:1]: Yes

File: http-python3.12-django5.0/Kraftfile (hunk 1 above)
Comment (mschlumpp) [reply http-python3.12-django5.0/Kraftfile:3]: Is there some explanation when to use one over the other?

File: http-python3.12-django5.0/README.md (hunk 2 above)
Comment (mschlumpp) [reply http-python3.12-django5.0/README.md:9]: Also for development you would use the `manage.py runserver` command.

File: http-python3.12-django5.0/Kraftfile (hunk 1 above)
Comment (nderjung) [reply http-python3.12-django5.0/Kraftfile:3]: We should only use the abbreviated version in examples, i.e. this change sans the registry. It makes this example compatible with OSS unikraft. The use of the full registry is an artifact of an older version of `kraft` which did not support such abbreviation.

File: http-python3.12-django5.0/README.md (hunk 2 above)
Comment (nderjung) [reply http-python3.12-django5.0/README.md:9]: We should just use `8080`. `80` is a restricted port on some hosts which require admin privileges and `8080` is common for local web development (along with `3000`).

Comment (mschlumpp) [reply http-python3.12-django5.0/README.md:9]: This limitation does not apply to docker or kraftcloud

**Pull request merged**